static/pages/partnerships/partnerships.html

```diff
@@ -285,61 +263,63 @@ <h3 class="md-display-1 partnerships-description-headline-padding">
           Building a dynamic, global community to transform online learning is 
           a bold vision that requires generous support.
         </p>
-        <h4 strong class="oppia-foundation-center-align-text">
+        <p class="oppia-foundation-center-align-text
+                  oppia-foundation-bold-text">
           That's why your help is so important.
-        </h4>
+        </p>
         <p class="oppia-foundation-center-align-text">
           Here are some of the challenges that your involvement can help us 
           tackle in the near future:
         </p>
       </md-content>
-      <div layout-gt-md="row"
+      <div class="oppia-foundation-vertical-padding"
+           layout-gt-xs="row"
            layout="column"
-           layout-align="center center" 
-           style="width: 75%;"
-           flex
-           class="oppia-foundation-vertical-padding">
-        <div flex="33"
-             layout="column"
-             layout-xs-align="end center"
+           layout-align="center center">
+        <div layout-gt-xs="column"
+             layout="row"
              layout-align="center center">
           <img src="/assets/icons/magnifier_icons.svg"
                class="partnerships-page-grow-card-icon"
                alt="Magnifier icon">
           <md-content class="oppia-foundation-white-background
                              oppia-foundation-black-text
+                             oppia-foundation-mobile-left-padding
                              partnerships-page-icon-description-text-box">
             <p class="oppia-foundation-center-align-text">
-              Conducting Randomized Controlled Trial
+              Conducting Controlled<br />Randomized Trial
```

Comment:
Here and below, just use `<br>`. We typically don't close void tags.

Better still, avoid using `<br>` altogether and use adjust the width of the container instead. Otherwise I think some screenreaders will read the break and that doesn't really make sense semantically.

Ditto below.

Reply:
You know, that makes way more sense than what I was trying to do with `<br>`, not sure what I was thinking there. 
Done!

Reply:
GIF for latest changes!
![2018-09-05 12 06 49](https://user-images.githubusercontent.com/11153258/45106314-52169f80-b104-11e8-82e1-d6f2e45a58d2.gif)